lib/anubis.go

```diff
@@ -344,7 +345,7 @@ func (s *Server) PassChallenge(w http.ResponseWriter, r *http.Request) {
 	// generate JWT cookie
 	token := jwt.NewWithClaims(jwt.SigningMethodEdDSA, jwt.MapClaims{
 		"challenge": challenge,
-		"nonce":     nonce,
+		"nonce":     nonceStr,
```

Comment:
I love how elegant this is.

Reply:
yeahh 😂 I originally started typing `strconv.Atoi(non` and then my auto suggest was like "nonceStr!!!"